app/models/video.rb

```diff
@@ -2,7 +2,7 @@ class Video < ActiveRecord::Base
   require 'YoutubeReader'
   require 'net/http'
 
-  attr_accessible :description, :title, :youtube_id, :starred, :duration
+  attr_accessible :description, :title, :youtube_id, :starred, :duration, :subject
```

Comment:
Subject isn't per se an accessible attribute, since it's an ActsAsTaggableOn model.